SwiftCheck/Test.swift

```diff
@@ -223,8 +223,10 @@ internal func quickCheckWithResult(args : Arguments, p : Testable) -> Result {
 					, randomSeed:			rnd()
 					, numSuccessShrinks:	0
 					, numTryShrinks:		0
-					, numTotTryShrinks:		0)
-	return test(state, p.property().unProperty.unGen)
+					, numTotTryShrinks:		0
+					, shouldAbort:			false)
+	let modP : Property = (p.exhaustive ? once(p.property()) : p.property())
```

Comment:
Oh wait, `Testable` is normal form?  Bummer that witnesses don't get expanded if I don't explicitly ask for a polymorphic constraint in `once`.